.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: release
+on:
+  push:
+    tags:
+      # this is a glob, not a regexp
+      - '[0-9]*'
+jobs:
+  source:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/cockpit-project/unit-tests
+      options: --user root
+    permissions:
+      # create GitHub release
+      contents: write
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      # https://github.blog/2022-04-12-git-security-vulnerability-announced/
+      - name: Pacify git's permission check
+        run: git config --global --add safe.directory /__w/anaconda-webui/anaconda-webui
+
+      - name: Workaround for https://github.com/actions/checkout/pull/697
+        run: git fetch --force origin $(git describe --tags):refs/tags/$(git describe --tags)
+
+      - name: Build release
+        run: make dist
+
+      - name: Publish GitHub release
+        uses: cockpit-project/action-release@88d994da62d1451c7073e26748c18413fcdf46e9
```

Comment:
I like this, however, couldn't it be versioned as other actions?

Reply:
It's custom action from cockpit. http://git.app.eng.bos.redhat.com/git/cockpit-release.git/

would be fine to use tags, but cockpit uses SHA as well, so I bet they don't tag this.

Reply:
Understand, would be nice but probably not requirement.